x-pack/legacy/plugins/apm/public/components/shared/charts/Timeline/VerticalLines.js

```diff
@@ -20,7 +20,9 @@ class VerticalLines extends PureComponent {
       tickValues
     } = this.props.plotValues;
 
-    const agentMarkTimes = this.props.agentMarks.map(({ us }) => us);
+    const markTimes = marks
+      .filter(mark => mark.docType === 'agentMark')
```

Comment:
I don't think we should specifically filter out agent marks. What about having a generic property on the mark type like `verticalLine: true` and then change this filter to:

```
.filter(mark => mark.verticalLine)
```